### PR TITLE
Add tags as list of text and build_url_html for grafana

### PR DESCRIPTION
**Pull request by ShauliSolomovich**
https://coralogix.atlassian.net/browse/PLATENG-1452

### CODE REVIEW

File: src/out [hunk 1]
```diff
@@ -91,6 +93,7 @@ body=$(jq -cn \
   --arg       timestamp "${timestamp}000" \
   --arg       severity "$severity" \
   --arg       build_url "$concourse_url/builds/$BUILD_ID" \
+  --arg       build_url_html "<a href=\"$concourse_url/builds/$BUILD_ID\">build_url</a>" \
```
Comment (ari-becker) [inline src/out:96]: just `build_url</a>` here?

Comment (ari-becker) [reply src/out:96]: Maybe `s/build_url/Link to Build`?

Comment (ari-becker) [reply src/out:96]: then, shouldn't `Link to Build` be configurable?

**Review by ari-becker — DISMISSED** [review]

**Review by ari-becker — APPROVED** [review]

File: src/out (hunk 1 above)
Comment (ShauliSolomovich) [reply src/out]: Chaned it to Link to  Build, i think its better, and its only for the grafana annotations to be able to be clickable

**Pull request merged**